plugins/DualFilter/DualFilterControlDialog.cpp

```diff
@@ -23,65 +23,63 @@
  *
  */
 
+#include "DualFilterControlDialog.h"
+
 #include <QLayout>
 
-#include "DualFilterControlDialog.h"
+#include "ComboBox.h"
 #include "DualFilterControls.h"
-#include "embed.h"
 #include "LedCheckbox.h"
-#include "ComboBox.h"
 #include "ToolTip.h"
+#include "embed.h"
 #include "gui_templates.h"
 
-#define makeknob( name, x, y, model, label, hint, unit ) 	\
-	Knob * name = new Knob( knobBright_26, this); 			\
-	name -> move( x, y );									\
-	name ->setModel( &controls-> model );					\
-	name ->setLabel( label );							\
-	name ->setHintText( hint, unit );
-
+#define makeknob(name, x, y, model, label, hint, unit) \
+	Knob* name = new Knob(knobBright_26, this); \
+	name->move(x, y); \
+	name->setModel(&controls->model); \
+	name->setLabel(label); \
+	name->setHintText(hint, unit);
 
-
-DualFilterControlDialog::DualFilterControlDialog( DualFilterControls* controls ) :
-	EffectControlDialog( controls )
+DualFilterControlDialog::DualFilterControlDialog(DualFilterControls* controls)
+	: EffectControlDialog(controls)
 {
-	setAutoFillBackground( true );
+	setAutoFillBackground(true);
 	QPalette pal;
-	pal.setBrush( backgroundRole(), PLUGIN_NAME::getIconPixmap( "artwork" ) );
-	setPalette( pal );
-	setFixedSize( 373, 109 );
+	pal.setBrush(backgroundRole(), PLUGIN_NAME::getIconPixmap("artwork"));
+	setPalette(pal);
+	setFixedSize(373, 109);
 
-	makeknob( cut1Knob, 24, 26, m_cut1Model, tr( "FREQ" ), tr( "Cutoff frequency" ), "Hz" )
-	makeknob( res1Knob, 74, 26, m_res1Model, tr( "RESO" ), tr( "Resonance" ), "" )
-	makeknob( gain1Knob, 124, 26, m_gain1Model, tr( "GAIN" ), tr( "Gain" ), "%" )
-	makeknob( mixKnob, 173, 37, m_mixModel, tr( "MIX" ), tr( "Mix" ), "" )
-	makeknob( cut2Knob, 222, 26, m_cut2Model, tr( "FREQ" ), tr( "Cutoff frequency" ), "Hz" )
-	makeknob( res2Knob, 272, 26, m_res2Model, tr( "RESO" ), tr( "Resonance" ), "" )
-	makeknob( gain2Knob, 322, 26, m_gain2Model, tr( "GAIN" ), tr( "Gain" ), "%" )
+	makeknob(cut1Knob, 24, 26, m_cut1Model, tr("FREQ"), tr("Cutoff frequency"), "Hz")
+		makeknob(res1Knob, 74, 26, m_res1Model, tr("RESO"), tr("Resonance"), "")
+			makeknob(gain1Knob, 124, 26, m_gain1Model, tr("GAIN"), tr("Gain"), "%")
+				makeknob(mixKnob, 173, 37, m_mixModel, tr("MIX"), tr("Mix"), "")
+					makeknob(cut2Knob, 222, 26, m_cut2Model, tr("FREQ"), tr("Cutoff frequency"), "Hz")
+						makeknob(res2Knob, 272, 26, m_res2Model, tr("RESO"), tr("Resonance"), "")
+							makeknob(gain2Knob, 322, 26, m_gain2Model, tr("GAIN"), tr("Gain"), "%")
```

Comment:
Since the semicolon is included in the `makeknob` macro, these are formatted as if they're nested in something, which I believe is unintended.

Reply:
Good catch! Terminating semicolons shouldn't be hidden inside macros. I'll fix it.

Reply:
Or perhaps `makeknob()` should not be a macro in the first place. A lambda that does what is needed and returns `Knob*` would be probably preferred in a modern C++ code. But that's a different refactor topic I guess.

Reply:
Closing this, since I think this is a code issue, not a formatting issue. Feel free to comment against it and reopen the conversation.